minard/HLDQTools.py

```diff
@@ -85,10 +85,15 @@ def import_TELLIEDQ_ratdb(runs):
         return runs, checkDict, runInformationDict
 
     for run in runs:
+        run_dict = {}
+        runInformation = {}
+        
         try:
-            run_dict = {}
             run_dict["fibre"] = data[run]["fibre"]
-            run_dict["pulse_delay"] = data[run]["pulse_delay"]
+            try:
```

Comment:
Before this line, could you please add a comment?
```Parameter pulse_delay was renamed to pulse_separation in RAT release 6.16.9```
The same for lines 120 and 133.